Fix problem with diversity algorithm causing disassocaition in 1.1.1

In 1.1.1 a patch went in which enables the "slow diversity" algorithm.
This algorithm forces one antenna or the other, if the background noise is
significantly quieter in one than the other.  It favors the quieter
antenna, and won't kick in unless the difference is significant.

This is showing to be problematic in some environments and is reverted
by this patch.

Signed-off-by: James Ketrenos <jketreno@linux.intel.com>

diff -Nupr ipw2200-1.1.1-orig/ipw2200.c ipw2200-1.1.1/ipw2200.c
--- ipw2200-1.1.1-orig/ipw2200.c	2006-03-08 06:42:55.000000000 -0600
+++ ipw2200-1.1.1/ipw2200.c	2006-03-08 14:32:20.000000000 -0600
@@ -10198,7 +10198,7 @@ static void init_sys_config(struct ipw_s
 	sys_config->disable_unicast_decryption = 1;
 	sys_config->exclude_multicast_unencrypted = 0;
 	sys_config->disable_multicast_decryption = 1;
-	sys_config->antenna_diversity = CFG_SYS_ANTENNA_SLOW_DIV;
+	sys_config->antenna_diversity = CFG_SYS_ANTENNA_BOTH;
 	sys_config->pass_crc_to_host = 0;	/* TODO: See if 1 gives us FCS */
 	sys_config->dot11g_auto_detection = 0;
 	sys_config->enable_cts_to_self = 0;
